src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -17,13 +22,49 @@ private Drivetrain() {
      * @param leftSpeed  Speed of left side.
      * @param rightSpeed Speed of right side.
      */
-    public void tankDrive(double leftSpeed, double rightSpeed) {
+    public void tankDrive(double leftPower, double rightPower)
+    {
+
+        if(!checkTankDriveNotBroken(leftPower,rightPower)){
+            this.stop();
+        }
     }
 
-    public static Drivetrain getInstance() {
-        return INSTANCE;
+    //this checks to make sure that if the robot is told to go forward, its actually traveling forward instead of backward
+    private boolean checkTankDriveNotBroken(double leftPower, double rightPower){
```

Comment:
This could use a more descriptive name, as well as proper JavaDoc (see `tankDrive` for an example.)